Cargo.toml

```diff
@@ -15,14 +15,17 @@ bitflags = "1"
 cty = "0.2.1"
 delog = "0.1.0"
 generic-array = "0.14"
-heapless = "0.7"
 
-[dependencies.cstr_core]
-default-features = false
-version = "0.2"
+littlefs-sys.workspace = true
+#  serde.workspace = true
+cstr_core.workspace = true
+heapless.workspace = true
+
+[target.'cfg(target_os = "espidf")'.dependencies]
+esp-idf-sys.workspace = true
 
-[dependencies.littlefs2-sys]
-version = "0.1.6"
+[build-dependencies]
+build-tools.workspace = true
```

Comment:
Because this crate is an external one, using workspace deps won't work (because this crate isn't a member of our workspace).

Reply:
good point.   I set this up when I thought it would be living in our juneoslite repo.